include/taskolib/Executor.h

```diff
@@ -27,6 +27,7 @@
 
 #include <future>
 #include <memory>
+#include <gul14/optional.h>
```

Comment:
I think this is a leftover? At least I do not see any use of `optional` anymore.